### PR TITLE
[All Hosts] (browsers) new item for Edge troubleshooting

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8500d52](https://github.com/OfficeDev/office-js-docs-pr/commits/8500d524a015e3a7daa95e77f1fd7f121c6d788b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-webview/docs/concepts/browsers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202003092206591468-1653%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=nxxNNmpqEbsqc0%2BMkHBqRoozMs4JnrE2NEZV5oEBdO4%3D&st=2020-03-09T22%3A03%3A10Z&se=2020-04-09T22%3A08%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[4f18957](https://github.com/OfficeDev/office-js-docs-pr/commits/4f1895794991ec3dda9ddb11c43ee7fc026e7c01)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-webview/docs/concepts/browsers-used-by-office-web-add-ins.md) | :warning:Warning | | [Details](#user-content-ffbf8e59-92e576b7)

<a id="ffbf8e59-92e576b7"></a>
### [docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-webview/docs/concepts/browsers-used-by-office-web-add-ins.md)
 - **Line 39**: **[Warning]** `````Invalid file link:(~/overview/office-add-ins).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202003092217434028-1653%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=MlsQ%2F%2B6rj8DrWRaSBrXYuBowt2BSNasghjst9qj%2B9Js%3D&st=2020-03-09T22%3A13%3A52Z&se=2020-04-09T22%3A18%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[98900b8](https://github.com/OfficeDev/office-js-docs-pr/commits/98900b8ea6d15a32ce671cda67751d4a8205331c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-webview/docs/concepts/browsers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202003092221189568-1653%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=JpbKs%2BvnYRrH53hoSoUb5aa%2BTClZH5yQUrqMX44aweQ%3D&st=2020-03-09T22%3A17%3A27Z&se=2020-04-09T22%3A22%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/concepts/browsers-used-by-office-web-add-ins.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ The following table shows which browser is used for the various platforms and op
 
 ## Troubleshooting Microsoft Edge Issues
 
+### Service Workers are not working
+
+Service Workers are not supported on [Microsoft Edge WebView](/microsoft-edge/hosting/webview) for Office add-ins. Please see [here](../overview/office-add-ins.md) for the latest supported features on the Edge Webview control. We are working hard to bring the new [Chromium based Edge WebView2](/microsoft-edge/hosting/webview2) to the Office add-ins platform, which we expect will support Service Workers.
```
Comment (ElizabethSamuel-MSFT) [inline docs/concepts/browsers-used-by-office-web-add-ins.md]:
```suggestion
Office Add-ins do not support Service Workers on [Microsoft Edge WebView](/microsoft-edge/hosting/webview). Please see the [Office Add-ins overview](../overview/office-add-ins.md) for the latest supported features on the Edge WebView control. We are working hard to bring the new [Chromium-based Edge WebView2](/microsoft-edge/hosting/webview2) to the Office Add-ins platform, which we expect will support Service Workers.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[67d6b97](https://github.com/OfficeDev/office-js-docs-pr/commits/67d6b977ffb2e671f025842b9b52fbdf1a6db0d2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-webview/docs/concepts/browsers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202003092233567756-1653%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=OVmtOWNEW1XS895vrlveRGUWvXUAwwDu1nni%2FafWybM%3D&st=2020-03-09T22%3A30%3A06Z&se=2020-04-09T22%3A35%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**